aikau/src/main/resources/alfresco/services/actions/CopyMoveService.js

```diff
@@ -124,10 +134,18 @@ define(["dojo/_base/declare",
                documents: documents,
                responseScope: payload.alfResponseScope
             };
+         var createLinkButtonLabel = payload.createLinkButtonLabel || "services.ActionService.createLink",
+             createLinkResponseTopic = this.generateUuid() + "_ALF_CREATE_LINK_LOCATION_PICKED",
+             createLinkPublishPayload = {
+                        nodes: nodes,
+                        documents: documents,
+                        responseScope: payload.alfResponseScope
+                     };
 
          var firstFileName = documents.length ? documents[0].fileName : null;
          var fileName = (nodes.length === 1 && firstFileName) ? firstFileName : this.message("services.ActionService.copyMoveTo.multipleFiles");
          this._actionHandle = this.alfSubscribe(responseTopic, lang.hitch(this, this.onCopyMoveLocationSelected, urlPrefix, payload.copy), true);
+         this._actionHandle = this.alfSubscribe(createLinkResponseTopic, lang.hitch(this, this.onCreateLinkLocationSelected), true);
```

Comment:
It looks like you're replacing the previous _actionHandle set here, although looking at the code that was already there it looks like there was a bug... the `onCopyMoveLocationSelected` function is trying to remove `_actionCopyHandle` which has never been set. It looks like we need to do a bit of tidying up here.

I would suggest that we make `_actionHandle` become an array called `_actionHandles` and then we push both subscription handles into it. The ensure that the handles are unsubscribed on an action.